webpack/assets/javascripts/react_app/components/ForemanModal/ForemanModalNoRedux.test.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
```

Comment:
Instead of `NoRedux` suffix we have a convention for tests files:
`Component.test.js` for unit test (without redux)
`integration.test.js` for react+redux test
`ComponentActions.test.js` for actions test
`ComponentReducer.test.js` for reducer test

Reply:
@amirfefer maybe you missed the [comment here](https://github.com/theforeman/foreman/pull/7105#discussion_r348710831).  The `ForemanModalNoRedux.test.js` is meant to replace `ForemanModal.test.js`. If you find it an acceptable approach I plan to rename it `ForemanModal.test.js`.

Reply:
@sharvit could you please look at this change of the test structure?